library/src/main/java/com/bumptech/glide/load/resource/drawable/ResourceDrawableDecoder.java

```diff
@@ -47,18 +47,27 @@ public boolean handles(@NonNull Uri source, @NonNull Options options) {
       @NonNull Options options) {
     @DrawableRes int resId = loadResourceIdFromUri(source);
     String packageName = source.getAuthority();
-    Context targetContext = packageName.equals(context.getPackageName())
-        ? context : getContextForPackage(source, packageName);
+    Context targetContext = getContextForPackage(source, packageName);
     // We can't get a theme from another application.
     Drawable drawable = DrawableDecoderCompat.getDrawable(context, targetContext, resId);
     return NonOwnedDrawableResource.newInstance(drawable);
   }
 
   @NonNull
   private Context getContextForPackage(Uri source, String packageName) {
```

Comment:
Probably rename to findContext since you're no longer just returning the context for the given package name.

Reply:
Done